benchmark/fs/bench-mkdtempSync.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const nonexistentDir = tmpdir.resolve('non-existent', 'foo', 'bar');
+const bench = common.createBenchmark(main, {
+  type: ['valid', 'invalid'],
+  n: [1e3],
+});
+
+function main({ n, type }) {
+  let prefix;
+
+  switch (type) {
+    case 'valid':
+      prefix = `${Date.now()}`;
+      break;
+    case 'invalid':
+      prefix = nonexistentDir;
+      break;
+    default:
+      new Error('Invalid type');
+  }
+
+  bench.start();
+  for (let i = 0; i < n; i++) {
+    try {
+      const folderPath = fs.mkdtempSync(prefix, { encoding: 'utf8' });
+      fs.rmdirSync(folderPath);
```

Comment:
Do we need to remove it here? I think as long as n does not exceed the number of combinations formed by the 6 random characters it should be fine to just leave them in the loop and let test common just delete the tmpdir on exit.